### PR TITLE
04

### PR DESCRIPTION
**Pull request by laurenfern**
Exercise 4 and extra credit 1-5

### CODE REVIEW

File: src/exercise/04.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Creating custom components
```
Comment (benjmyers) [inline src/exercise/04.md]: Guessing a fresh pull of `main` to bring your branch up to date will get rid of these changes you didn't make.

File: src/exercise/04.html [hunk 2]
```diff
@@ -6,19 +6,30 @@
...
+    }
+
     const element = (
-      <div className="container">
-        <div className="message">Hello World</div>
-        <div className="message">Goodbye World</div>
-      </div>
+      <>
+        <Message subject="World" greeting="Hello Fragment"/>
+        <Message subject="World" greeting="Goodbype Fragment"/>
```
Comment (benjmyers) [inline src/exercise/04.html]: Typo but 🤷

**Review by benjmyers — APPROVED** [review]
🎉

File: src/exercise/04.html (hunk 2 above)
Comment (laurenfern) [reply src/exercise/04.html]: 👍

**Pull request merged**